app/engines/v2_api.rb

```diff
@@ -143,6 +143,7 @@ class V2Api < ::Rails::Engine
       get 'goods_nomenclatures/chapter/:chapter_id', to: 'goods_nomenclatures#show_by_chapter', constraints: { chapter_id: /\d{2}/ }
       get 'goods_nomenclatures/heading/:heading_id', to: 'goods_nomenclatures#show_by_heading', constraints: { heading_id: /\d{4}/ }
       get 'goods_nomenclatures/:id', to: 'goods_nomenclatures#show', constraints: { id: /\d{4,10}/ }
+      post 'goods_nomenclatures/batch', to: 'goods_nomenclatures#show_by_batch'
```

Comment:
If we're limiting to 10 commodity codes we could probably just put the codes in the query param and use a GET?

Also, isn't this basically an index action with a filter?